WcaOnRails/app/models/competition.rb

```diff
@@ -27,7 +27,7 @@ class Competition < ApplicationRecord
            allow_nil: true,
            with_model_currency: :currency_code
 
-  scope :not_over, -> { where("end_date >= ?", Date.today) }
+  scope :not_over, -> { where("end_date >= ? AND results_posted_at IS NULL", Date.today) }
```

Comment:
Gah, I *hate* keeping these scopes in sync with our boolean model methods. @larspetrus, any ideas to help us not repeat ourselves here?